### PR TITLE
Changelog update for 7.53.0 release

### PR DESCRIPTION
**Pull request by kacper-murzyn**

### CODE REVIEW

**Review by Pythyu — APPROVED** [review]

File: CHANGELOG-DCA.rst [hunk 1]
```diff
@@ -2,6 +2,44 @@
 Release Notes
 =============
 
+.. _Release Notes_7.53.0:
+
+7.53.0 / 6.53.0
+======
```
Comment (olivielpeau) [inline CHANGELOG-DCA.rst:8]: Note: this seems to break formatting of titles (this was fixed in `main` but not on the 7.53.x branch, seems like the changelog generation still generates wrongly-formatted titles)

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,137 @@
 Release Notes
 =============
 
+.. _Release Notes_7.53.0:
+
+7.53.0 / 6.53.0
+======
```
Comment (olivielpeau) [inline CHANGELOG.rst:8]: ditto

File: CHANGELOG.rst [hunk 3]
```diff
@@ -2,6 +2,137 @@
...
+  whether the kubelet core check should be loaded.
+
+- Added LastSuccessfulTime to cronjob status payload.
+
+- Add a retry mechanism to Software Bill of Materials (SBOM) collection for container images.
+  This will help to avoid intermittent failures during the collection process.
+
+- Add startup timestamp to the Agent metadata payload.
+
+- Agents are now built with Go ``1.21.8``.
```
Comment (olivielpeau) [inline CHANGELOG.rst:63]: nit: remove this entry (semantically, it should be replaced by the next entry about Go 1.21.9)

**Review by olivielpeau — APPROVED** [review]
A few nits, feel free to address now or in a subsequent PR

**Pull request merged**